libs/coin-framework/src/bot/types.ts

```diff
@@ -147,6 +151,11 @@ export type AppSpec<T extends TransactionCommon> = {
   allowEmptyAccounts?: boolean;
   // do not keep operations in accounts (Cosmos family case)
   skipOperationHistory?: boolean;
+  onSpeculosDeviceCreated?: (device: {
```

Comment:
add an inline comment as other methods in this files plz

Reply:
done